WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormActionsFactory.swift

```diff
@@ -208,6 +219,8 @@ private extension ProductFormActionsFactory {
             return product.product.categories.isNotEmpty
         case .tags:
             return product.product.tags.isNotEmpty
+        case .linkedProducts:
+            return isEditProductsRelease5Enabled && (product.upsellIDs.count > 0 || product.crossSellIDs.count > 0)
```

Comment:
I think we have `isNotEmpty`. 🙂 

```suggestion
            return isEditProductsRelease5Enabled && (product.upsellIDs.isNotEmpty || product.crossSellIDs.isNotEmpty)
```